examples/00-postprocessing/post_processing_legacy.py

```diff
@@ -20,15 +20,29 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-""".. _ref_post_processing_exhaust_manifold:
-
-Postprocessing using PyVista and Matplotlib
--------------------------------------------
-This example uses PyVista and Matplotlib to demonstrate PyFluent
-postprocessing capabilities. The 3D model in this example
-is an exhaust manifold that has high temperature flows passing
-through it. The flow through the manifold is turbulent and
-involves conjugate heat transfer.
+""".. _ref_post_processing_legacy:
+
+Post-processing using PyVista and Matplotlib (Legacy)
+-----------------------------------------------------
+Demonstrates legacy post-processing capabilities in PyFluent using
+the PyVista and Matplotlib interfaces.
+
+This example loads an exhaust manifold case and data file and performs a series
+of visualization and analysis tasks. It showcases how to create and display
+various graphical and plot-based outputs including meshes, plane and iso
+surfaces, contours, vectors, pathlines, and XY plots. It also illustrates
+residual and monitor plotting using Matplotlib.
+
+Key aspects illustrated in this example include:
+- Use of the ``Graphics`` and ``Plots`` interfaces for visualization and analysis.
+- Creation of 3D geometric entities such as plane and iso surfaces.
+- Generation of contour and vector plots on selected regions.
+- Setup and display of residual and monitor plots during solver iterations.
+- Execution of solver iterations with hybrid initialization and live plot updates.
+
+This example provides a comprehensive demonstration of traditional
+post-processing workflows available in PyFluent prior to the new
+interactive visualization API.
```

Comment:
A couple of concerns about this description:

“Traditional vs. new” assumes users know or care about the evolution of our APIs.
Most users are looking for the recommended way to work today, not a historical contrast.

Including a “legacy” example may be counterproductive.
If the new visualization API exists because we believe it is better, then highlighting an older workflow raises questions:

- Do users need to learn both?

- Are we implicitly committing to maintain the older style indefinitely?

- Does this make future deprecation harder?

Would it be better to reframe this example around what it does, rather than positioning it as “traditional”? Or, if we truly intend it to be a legacy reference, should we clearly mark it as such and explain why it still exists?